src/app/components_ngrx/work-item-quick-add/work-item-quick-add.component.ts

```diff
@@ -147,6 +149,8 @@ export class WorkItemQuickAddComponent implements OnInit, OnDestroy, AfterViewIn
     if (event) {
       event.preventDefault();
     }
+    let currentRoute = cloneDeep(this.route.snapshot.queryParams);
+    let selectedIterationID = this.filterService.getConditionFromQuery(currentRoute.q, 'iteration');
```

Comment:
If we have multiple iterations in the same query then this will likely fail.

Reply:
@sanbornsen  Can we have multiple iterations in the query? If yes, then which iterations wis will be shown in the list?

Reply:
Since currently one wi can be associated to a single iteration then to which iteration the newly created wi will be added if we have multiple iterations in the query.